### ISSUE TITLE
Python xlo.func() wrapped functions not callable

### ISSUE THREAD

**Issue by andyvandy** [cunnane/xloil#58]
### Example
I'm not able to call wrapped functions, I'd expect the following to pass:
```python
import xloil as xlo

def example():
    return "HELLO"

@xlo.func()
def example_wrapped():
    return "WORLD"
    
def test_callable():
    assert example() == "HELLO"
    assert example_wrapped() == "WORLD"
```

<details>
  <summary>Error logs</summary>

```
self = <xloil.register._WorksheetFunc object at 0x037814B0>, args = ()
kwargs = {}

def __call__(self, *args, **kwargs):
\>     return self.func(*args, **kwargs)
E       AttributeError: '_WorksheetFunc' object has no attribute 'func'

.venv\Lib\site-packages\xloil\register.py:241: AttributeError
```
</details>

I believe this snippet could be worked into a simple test case

### Suggested Fix
We change `__call__` in `_WorksheetFunc` ([register.py:L241](https://github.com/cunnane/xloil/blame/eb710e877270a7cbaf2cbc9928edb26ba09baac9/libs/xlOil_Python/Package/xloil/register.py#L241 ))
```python
def __call__(self, *args, **kwargs):
        return self.func(*args, **kwargs)
```
to 
```python
def __call__(self, *args, **kwargs):
        return self.__wrapped__ (*args, **kwargs)
```
I did a quick sweep and didn't see an alternative code path which was setting func explicitly

**Comment by cunnane** [comment]
Thanks for reporting this and providing the fix!  This is incorporated into 0.17.13 which was released this week

**Comment by andyvandy** [comment]
Amazing, thanks for accepting and comiting the proposal!
